milli/Cargo.toml

```diff
@@ -98,16 +98,7 @@ rand = { version = "0.8.5", features = ["small_rng"] }
 
 [features]
 all-tokenizations = [
-    "charabia/chinese",
-    "charabia/hebrew",
-    "charabia/japanese",
-    "charabia/thai",
-    "charabia/korean",
-    "charabia/greek",
-    "charabia/khmer",
-    "charabia/vietnamese",
-    "charabia/swedish-recomposition",
-    "charabia/german-segmentation",
+    "charabia/default",
```

Comment:
is this equivalent to the previous explicit list?

Reply:
Hello @dureuill, yes I checked the default Charabia's feature and it's the same, I changed to default to avoid forgetting a feature when updating Charabia in the future